posix/include/sof/lib/dma.h

```diff
@@ -41,6 +41,12 @@ struct comp_buffer;
  */
 
 /* DMA direction bitmasks used to define DMA copy direction */
+#define SOF_DMA_DIR_MEM_TO_MEM		DMA_DIR_MEM_TO_MEM
+#define SOF_DMA_DIR_HMEM_TO_LMEM	DMA_DIR_HMEM_TO_LMEM
+#define SOF_DMA_DIR_LMEM_TO_HMEM	DMA_DIR_LMEM_TO_HMEM
+#define SOF_DMA_DIR_MEM_TO_DEV		DMA_DIR_MEM_TO_DEV
+#define SOF_DMA_DIR_DEV_TO_MEM		DMA_DIR_DEV_TO_MEM
+#define SOF_DMA_DIR_DEV_TO_DEV		DMA_DIR_DEV_TO_DEV
```

Comment:
ok, maybe the compiler doesn't care, but wouldn't it be easier for the humans to have these under right-hand definitions just below? More cases below

Reply:
@lyakh Hmm, ok, let me make a rare concession for the humans among us and change the order. Addressed in the updated version.

Reply:
Oops, sorry, I made the changes to xtos/lib/dma.h. Let me make the same to posix/lib/dma.h as well. Will make a new update in a minute.